contracts/utils/contracts/test/TestOwnable.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.9;
+
+import "../src/Ownable.sol";
+
+
+contract TestOwnable is
+    Ownable
+{
+    function externalOnlyOwner()
+        external
+        onlyOwner
+    {} // solhint-disable-line no-empty-blocks
```

Comment:
It may be good to return something on success to avoid false positives. Perhaps the function selector, although just returning a bool is probably fine too.